boost_histogram/_internal/hist.py

```diff
@@ -406,5 +405,5 @@ def __getitem__(self, index):
             )
 
     def __setitem__(self, index, value):
-        indexes = _compute_commonindex(self._hist, index, expand=True)
+        indexes = _compute_commonindex(self._hist, index, expand=False)
```

Comment:
Expand expands `...`. It may go away or change drastically when we add proper set support next; but for now, you can't expand when you only support setting a single value at a time. :)

Reply:
(Note to self: Given that neither Hans nor I knew what expand meant (and I wrote it), we should rename to `expand_ellipsis`)

Reply:
OK